### ISSUE TITLE
Rename Human theme to Mint theme

### ISSUE THREAD

**Issue by Yanpas** [linuxmint/libreoffice-style-mint#1]
Benefits:
* No confusion between standart ubuntu theme
* Possibility to install both themes
* Simplicity to rename (just rename symlink and archive name, LO automatically display new theme in settings)
* No downsides with userconf (LO automatically switches to first availible theme, if can't find previous)

![_2015-11-09_17-03-09](https://cloud.githubusercontent.com/assets/7345761/11035559/47814408-8704-11e5-8a6c-2bceb041e19f.png)

I will try to create PR

**Comment by philipzae** [comment]
Yes LibreOffice 5.0 and above will list all icon theme .zip files in the /usr/lib/libreoffice/share/config/ and Ubuntu 16.04 will no longer be shipping images_human.zip as the human theme is being deprecated in favour of breeze.
